spec/javascripts/components/story/story_actions_spec.js

```diff
@@ -0,0 +1,47 @@
+import jasmineEnzyme from 'jasmine-enzyme';
+import React from 'react';
+import { shallow } from 'enzyme';
+import StoryItem, {
+  StoryPoints,
+  StateActions,
+  StateButton,
+  StoryActionFor
+} from 'components/story/StoryItem';
+
+let props = {};
+let wrapper = {};
+
+describe('<StateActions />', () => {
+  describe('When estimate is null',() => {
+    it("renders <StoryPoints /> component", () => {
+      const props = {
+        storyType: "feature",
+        estimate: null,
+        state: "started"
+      };
+      const wrapper = shallow(<StateActions {...props} />);
+      expect(wrapper.find(StoryPoints)).toExist();
+    });
+  });
+  describe('When estimate is not null', () => {
+    const states = [
+      "started",
+      "finished",
+      "delivered",
+      "rejected",
+      "accepted",
+      "unstarted"
+    ];
+    states.forEach((state) => {
+      describe(`When state = ${state}`, () => {
+        it('renders the <StateButton /> component',() => {
+          const wrapper = shallow(<StateActions state={state} estimate={1} storyType="feature" />);
+          const actions = StoryActionFor[state]
+          actions.forEach((action) => {
+            expect(wrapper.find(`StateButton[action="${action}"]`).exists()).toEqual(true);
```

Comment:
I think we can also improve this spec :thinking: 

```suggestion
            expect(wrapper.find(`StateButton[action="${action}"]`)).toExist();
```

Reply:
Spec improved.